File: noop-java/src/main/java/com/exyui/android/debugbottle/ui/BlockCanary.java
package com.exyui.android.debugbottle.ui;

import android.util.Log;

/**
 * Created by yuriel on 9/7/16.
 */

public class BlockCanary {
    public static final BlockCanary INSTANCE = new BlockCanary();
    private BlockCanary() {}
    private static final String TAG = "FakeBlockCanary";
    public BlockCanary install(Object a) {
        Log.d(TAG, "install");
        return this;
    }

    public void start() {
        Log.d(TAG, "start");
    }
}

File: noop/src/main/kotlin/com/exyui/android/debugbottle/ui/BlockCanary.kt
package com.exyui.android.debugbottle.ui

import android.util.Log

/**
 * Created by yuriel on 8/30/16.
 */
object BlockCanary {
    private val TAG = "FakeBlockCanary"
    fun install(any: Any): BlockCanary {
        Log.d(TAG, "install")
        return this
    }

    fun start() {
        Log.d(TAG, "start")
    }
}